src/transformers/models/bamba/configuration_bamba.py

```diff
@@ -186,6 +189,7 @@ def __init__(
         self.mamba_chunk_size = mamba_chunk_size
         self.mamba_conv_bias = mamba_conv_bias
         self.mamba_proj_bias = mamba_proj_bias
+        self.zloss_coeff = z_loss_coefficient
```

Comment:
```suggestion
        self.z_loss_coefficient = z_loss_coefficient
```
Let's propogate it everywhere 👀 (also in modular file etc)

Reply:
ah ok